transport/abstraction.py

```diff
@@ -2,16 +2,22 @@
 from typing import Any
 
 from raiden.message_handler import MessageHandler
-from raiden.messages import Message
 from raiden.raiden_service import RaidenService
-from raiden.transfer.identifiers import QueueIdentifier
 from raiden.utils import Address
+from transport.components import Message
 
 
-class TransportLayer(ABC):
+class Layer(ABC):
```

Comment:
Mahybe Layer is too generic as a name, because if you think this you could ask, why a type called Layer have send_message as a member? Maybe you can use something a little more specific here like CommunicationsLayer or something else.

Reply:
it's already in the transport package, so it's actually `transport.abstractions.layer`

Reply:
also, i made sure to used named imports elsewhere to avoid confusion